src/codecs/jats/index.ts

```diff
@@ -381,9 +383,60 @@ function decodeFront(
         meta: decodeMetaFront(front),
         pageStart: decodePageStart(front),
         pageEnd: decodePageEnd(front),
+        ...(decodeAbout(front)?.length && { about: decodeAbout(front) }),
+        genre: decodeGenres(front),
```

Comment:
There is quite a lot of repetition here, with `all(front, 'subj-group')` being called three times. How about, doing this instead?

Get all the `<subj-group>` elements:

```ts
const subjGroups = all(front, 'subj-group')
```
and then pass them through to the `decodeKeyword`, `decodeAbout` and `decodeGenres` functions:

```ts
        keywords: decodeKeywords(front, subjGroups),
        ...
        about: decodeAbout(subjGroups),
        genre: decodeGenres(subjGroups),
}
```

Also, ensure that `decodeKeyword`, `decodeAbout` and `decodeGenres` all return `undefined` if the array length == 0.